java/client/test/org/openqa/selenium/chrome/DevToolsInfrastructureTest.java

```diff
@@ -0,0 +1,28 @@
+package org.openqa.selenium.chrome;
+
+import org.junit.After;
+import org.junit.Before;
+import org.openqa.selenium.devtools.DevTools;
+
+
+public class DevToolsInfrastructureTest {
```

Comment:
Done. 

Added DevTools abstract class in devtools package. In addition, I made it generic so that only Chrome DevTools tests will run in "test_chrome" and we have the infra for the Edge in the future